coalib/results/Result.py

```diff
@@ -25,7 +26,7 @@
                    'aspect',
                    'additional_info',
                    'debug_msg')
-class Result:
+class Result(Comparables):
```

Comment:
We want to register `Result` to be a deriving type of `Comparables` inside the `generate_eq`/`generate_ordering` decorators, not here :+1: